deploy/kubernetes/deploy-driver.sh

```diff
@@ -95,6 +96,36 @@ fi
 ${KUBECTL} version
 
 readonly tmp_spec=/tmp/gcp-compute-persistent-disk-csi-driver-specs-generated.yaml
-${KUSTOMIZE_PATH} build ${PKGDIR}/deploy/kubernetes/overlays/${DEPLOY_VERSION} | tee $tmp_spec
-${KUBECTL} apply -v="${VERBOSITY}" -f $tmp_spec
 
+if [[ ${DEPLOY_OS} = "mixed" ]]; then
```

Comment:
I would suggest to simplify things and just always include both linux and windows node specs. I guess we may have a problem in the beginning where the windows spec is not available for staging/stable.

What if the kustomization.yaml in  image_setup/{dev/staging/stable} apppropriately pulls in node_base/windows as needed?

Then we don't actually need to auto-generate a top level kustomization.yaml and can continue to use the appropriate image_setup/X/kustomization.yaml

Reply:
At first, I thought with multiple bases, it is easy to have arbitrary combination, e.g. linux + stable_image, windows + alpha_image. But the image part does not work as expected in multiple bases. I will simply this with one combined driver instead of having a complicated script here.